content/cloud/changelog/cloud-config.md

```diff
@@ -0,0 +1,23 @@
+title = "Fermyon Cloud Variables Configuration"
+template = "changelog_item"
+date = "2022-06-13T07:00:00Z"
+enable_shortcodes = true
+tags = ["storage"]
```

Comment:
I would recommend adding a new tag here... perhaps "config" or "variable." @karthik2804 is there any additional plumbing we need to do if we add a new tag?

Reply:
Nope! just adding it there should work.